recipes/mmlong2-lite/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "mmlong2-lite" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "66cc1063df513697cafb524698e4125a5cffa2e63f80c209d10ee3e041049601" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Serka-M/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('mmlong2-lite', max_pin="x.x") }}
+
+requirements:
+  run:
+    - snakemake 8.2.3
+    - singularity 3.8.6
+    - zenodo_get 1.6.1
+    - pv 1.6.6
+    - pigz 2.6
+    - tar 1.34
+
+test:
+  commands:
+    - which mmlong2-lite
+
+about:
+  home: "https://github.com/Serka-M/mmlong2-lite"
+  license: GPL3
```

```suggestion
  license: GPL-3.0-only
```
or `GPL-3.0-or-newer`
Please see [SPDX](https://spdx.org/licenses/)